src/components/string_input_wrapper.tsx

```diff
@@ -17,7 +20,7 @@ interface StringInputWrapperProps {
  * @param path The path (consistent with the path provided to the lodash
  * get() method) to the datum within the data store for the StringInput component
  */
-const StringInputWrapper: FC<StringInputWrapperProps> = ({label, path}) => {
+const StringInputWrapper: FC<StringInputWrapperProps> = ({label, path, min=1, max=1000, regexp=/.*/}) => {
```

Comment:
Set `min=0`, which will allow the input to be optional.

Also, `max=1024` would be a more computer sciency value.